<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudioPlayer</class>
 <widget class="QMainWindow" name="AudioPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Audio Player</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background: transparent;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutMain">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutHeader">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #ff5555;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #ff0000;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMinimize">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #ffbb33;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #ff8800;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelTrackName">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18pt; 
    font-weight: 600;
    color: white;
    background-color: rgba(45, 45, 77, 180);
    border-radius: 15px;
    padding: 10px;
    font-family: "Montserrat", sans-serif;
</string>
      </property>
      <property name="text">
       <string>Трек не загружен</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="sliderPosition">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>10</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    height: 6px;
    background: #535353;
    border-radius: 3px;
}
QSlider::handle:horizontal {
    background: #ffffff;
    border: none;
    width: 14px;
    height: 14px;
    margin: -4px 0;
    border-radius: 7px;
}
QSlider::sub-page:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #6a11cb, stop:1 #2575fc);
}</string>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTotalTime">
      <property name="styleSheet">
       <string notr="true">font-size: 12px;
color: #b3b3b3;
font-family: &quot;Montserrat&quot;, sans-serif;</string>
      </property>
      <property name="text">
       <string>00:00 / 00:00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutControls">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnPrev">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 20pt;
border: 2px solid #9c27b0;
border-radius: 25px;
background-color: #2d2d4d;
color: #b388ff;
font-family: &quot;Montserrat&quot;, sans-serif;
font-weight: 600</string>
        </property>
        <property name="text">
         <string>PREV</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPlayPause">
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>70</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 28pt;
border: 3px solid #9c27b0;
border-radius: 35px;
background-color: #2d2d4d;
color: #9c27b0;
font-weight: 650;
font-family: &quot;Montserrat&quot;, sans-serif;
padding: 5px;</string>
        </property>
        <property name="text">
         <string>PLAY</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnNext">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font-size: 20pt;
border: 2px solid #9c27b0;
border-radius: 25px;
background-color: #2d2d4d;
color: #b388ff;
font-family: &quot;Montserrat&quot;, sans-serif;
font-weight: 600</string>
        </property>
        <property name="text">
         <string>NEXT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QListWidget {
    background-color: rgba(45, 45, 77, 150);
    border: 1px solid #535353;
    border-radius: 10px;
    color: #ffffff;
    font-size: 14px;
    padding: 5px;
    font-weight: 550;
}
QListWidget::item {
    padding: 8px;
    border-bottom: 1px solid #3d3d5c;
}
QListWidget::item:hover {
    background-color: rgba(106, 17, 203, 100);
}
QListWidget::item:selected {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #6a11cb, stop:1 #2575fc);
    color: #ffffff;
    border-radius: 5px;
}
QScrollBar:vertical {
    border: none;
    background: transparent;
    width: 8px;
    margin: 0px;
}
QScrollBar::handle:vertical {
    background: #6a11cb;
    min-height: 20px;
    border-radius: 4px;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="sliderVolume">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    height: 6px;
    background: #535353;
    border-radius: 3px;
}
QSlider::handle:horizontal {
    background: #ffffff;
    border: none;
    width: 14px;
    height: 14px;
    margin: -4px 0;
    border-radius: 7px;
}
QSlider::sub-page:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #6a11cb, stop:1 #2575fc);
}</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
